### DISCUSSION TITLE
expense setup using ansible

### DISCUSSION THREAD

**Discussion by Bhashitha19** [learndevopsonline/learndevopsonline#1881] [Q&A]
when iam trying to add expenses it will not add and show the expenses
![Screenshot (43)](https://github.com/learndevopsonline/learndevopsonline/assets/148459705/cbc0e890-464f-44d5-bd57-dc28705e8054)

**Comment by chaithanya1812** [comment]
### you backend service is not running
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/f5a126bb-9f29-4ab5-b8d0-e50b493a58de)

**Reply by Bhashitha19** [reply]
![Screenshot (44)](https://github.com/learndevopsonline/learndevopsonline/assets/148459705/0103680e-10f8-4ff4-81ab-cbdbf6ea9cb3)

**Comment by Bhashitha19** [comment]
still it doesn't  add expenses
![Screenshot (45)](https://github.com/learndevopsonline/learndevopsonline/assets/148459705/2399c7c7-9485-402e-ae75-7af1bbf17fa9)

**Comment by chaithanya1812** [comment]
### How many frontend instances you are running, browser-IP and instance-IP are different.

## Issue check once this was your mysql private-ip address, I am unable to login?
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/7f3114fa-e420-4425-9c2a-3cc48004a33d)

**Comment by chaithanya1812** [comment]
Hlo @Bhashitha19 playbook successfully run in new server,

Update the  mysql record  in route53 with these new mysql ip. 

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/1d46d6dc-57f8-4846-9cda-8ed22b4261bb)
